### ISSUE TITLE
Anisette server data changing on reboot

### ISSUE THREAD

**Issue by PandaPootchy** [Dadoum/Provision#42]
So I am using the docker version of this to just use the anisette server for altserver for linux since the anisette data im using didnt work since i removed the device from my apple id and the anisette server that is provided in the documentation dont work with arm. So I was wondering is there a way I can make the data stay the same after rebooting because when it changes altstore wants me to login since apple thinks im logging into another device.

**Comment by Dadoum** [comment]
Yes, anisette_server has a flag to preserve machine identification data across launches. Use `./anisette_server -r=true`.

**Comment by PandaPootchy** [comment]
How would I set that with the docker container because thats how I have it running since when I tried to run just using the file I got this error.
![image](https://user-images.githubusercontent.com/20006777/210422021-747c7599-0577-447d-84ff-3c4f0e8ee7c3.png)

**Comment by PandaPootchy** [comment]
So I got the file to work and tried with it but It still is not saving the machine identification since altstore is still making my log in everytime I restart the anisette server.

**Comment by PandaPootchy** [comment]
So was looking into more of it and come to find out it was saving the data when i did --remember-machine. now the strange part is when I would ctrl c to stop the server to restart it to see if it was saving it was like a hit and miss. sometimes it would make me relog in to altstore or not. I started just letting it run and restarting my server its self (simulating a weekly restart) and I did it 3 times and them 3 times it hasnt asked me to relog in. Not for sure what could be causing that but will go ahead and close this issue since it is kinda solved.
